packages/core/src/internal/execution/jsonrpc-client.ts

```diff
@@ -640,7 +640,7 @@ export class EIP1193JsonRpcClient implements JsonRpcClient {
     // We prioritize EIP-1559 fees over legacy gasPrice fees, however,
     // polygon (chainId 137) requires legacy gasPrice fees so we skip EIP-1559 logic in that case
     if (latestBlock.baseFeePerGas !== undefined && chainId !== 137) {
-      if (latestBlock.baseFeePerGas === 0n) {
+      if (latestBlock.baseFeePerGas === 0n && chainId != 56) {
```

Comment:
```suggestion
      if (latestBlock.baseFeePerGas === 0n && chainId !== 56) {
```

Reply:
This will satisfy our eslint.

Reply:
Thank you for letting me know. I should have read contribution guide carefully. Today was the contract deployment day. We handled this problem with downgrading version to v0.15.0.

Reply:
Have you been able to test the change with a deployment against BNB?

Reply:
yes. by modifying the line, I confirmed that BNB distribution was successful.

this is original stacktrace.
```
√ Confirm deploy to network bnb (56)? ... yes
Hardhat Ignition 🚀

Deploying [ TestTokenModule ]

Batch #1
  Executing TestTokenModule#TestToken...

An unexpected error occurred:

ProviderError: transaction underpriced: tip needed 1000000000, tip permitted 0
    at HttpProvider.request (node_modules\hardhat\src\internal\core\providers\http.ts:90:21)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async EIP1193JsonRpcClient.sendTransaction (node_modules\@nomicfoundation\ignition-core\src\internal\execution\jsonrpc-client.ts:418:24)
    at async sendTransactionForOnchainInteraction (node_modules\@nomicfoundation\ignition-core\src\internal\execution\future-processor\helpers\network-interaction-execution.ts:181:18)
    at async sendTransaction (node_modules\@nomicfoundation\ignition-core\src\internal\execution\future-processor\handlers\send-transaction.ts:86:18)
    at async FutureProcessor.processFuture (node_modules\@nomicfoundation\ignition-core\src\internal\execution\future-processor\future-processor.ts:114:9)
    at async ExecutionEngine._executeBatch (node_modules\@nomicfoundation\ignition-core\src\internal\execution\execution-engine.ts:153:30)
    at async ExecutionEngine.executeModule (node_modules\@nomicfoundation\ignition-core\src\internal\execution\execution-engine.ts:114:25)
    at async Deployer.deploy (node_modules\@nomicfoundation\ignition-core\src\internal\deployer.ts:194:25)
    at async SimpleTaskDefinition.action (node_modules\@nomicfoundation\hardhat-ignition\src\index.ts:302:24)
```

after the change
```
√ Confirm deploy to network bnb (56)? ... yes
Hardhat Ignition 🚀

Resuming existing deployment from .\ignition\deployments\chain-56

Deploying [ TestTokenModule ]

Batch #1
  Executed TestTokenModule#TestToken

Batch #2
  Executed TestTokenModule#TestToken.mint

[ TestTokenModule ] successfully deployed 🚀

Deployed Addresses

TestTokenModule#TestToken - (hide)
```